endpointmanager/pkg/nppesquerier/populatenpicontacts.go

```diff
@@ -110,8 +161,13 @@ func readContactCsv(ctx context.Context, filename string) ([][]string, error) {
 		// ok
 	}
 
+	err, newfilename := removeNestedDoubleQuotesFromCSV(filename)
```

Comment:
should this clean up/delete the created file at the end?

Reply:
i considered that, but i had found it useful to have a copy of a "correct" CSV on hand, might also be useful in the future to aid in debugging should any problems arise. Fortunately the contact file is one of the smaller ones included in the NPPES package.